src/Components/CurrentWeather.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import partlycloudy from "../img/weather-icons/partlycloudy.svg";
+
+const CurrentWeather = () => {
+  return (
+    <div className="Todaysweather">
+      <div className="currentWeather">
+        <img className="currentoverview" src={partlycloudy} />
+        <div className="overview">overcast clouds</div>
+        <div className="temperature">
+          <span className="data-heading"> Temperature</span> 10 to 11c
```

Comment:
You're missing the correct "degree symbol" after the temperature. You can simply google for it and copy-paste it.